From: Stuart Prescott <stuart@debian.org>
Date: Tue, 10 Sep 2024 06:34:52 +1000
Subject: Mark arm64 test failures as xfail temporarily

Try to at least get a package built to be able to debug the failure
---
 build_history/blacklist.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

Index: pyside6/build_history/blacklist.txt
===================================================================
--- pyside6.orig/build_history/blacklist.txt
+++ pyside6/build_history/blacklist.txt
@@ -91,19 +91,3 @@
 [QtQml::qqmlincubator_incubateWhile]
     pypy        # gives timeout on macOS if I run video converter while testing
 # 2022-03-04 PyPy: Currently 28 of 657 tests fail
-
-# Debian packaging failing tests
-[sample::array_numpy]
-    py3
-[sample::nontypetemplate]
-    py3
-[pyside6-android-deploy::test_pyside6_android_deploy]
-    py3
-[QtCore::qinstallmsghandler_test]
-    py3
-[QtCore::loggingcategorymacros_test]
-    py3
-[QtGui::qpainter_test]
-    py3
-[QtCharts::qcharts_numpy_test]
-    py3
